build/deps.mk

```diff
@@ -51,9 +52,27 @@ THRIFT = $(BIN)/thrift
 PROTOC_LIB = $(LIB)/protoc-$(PROTOC_VERSION)
 PROTOC_ZIP = $(PROTOC_LIB)/protoc.zip
 PROTOC = $(BIN)/protoc
-GEN_BINS = $(THRIFT) $(PROTOC)
+GEN_BINS = $(THRIFT) $(PROTOC) $(RAGEL)
 EXTRA_BINS = $(GLIDE)
 
+RAGEL_TAR = $(LIB)/ragel-$(RAGEL_VERSION)/ragel-$(RAGEL_VERSION).tar.gz
+RAGEL = $(BIN)/ragel
+
+$(RAGEL_TAR):
+	@mkdir -p $(dir $(RAGEL_TAR))
+	curl -L "https://www.colm.net/files/ragel/ragel-$(RAGEL_VERSION).tar.gz" > $(RAGEL_TAR)
+
+$(RAGEL): $(RAGEL_TAR)
+	@mkdir -p $(BIN)
+	@cd $(dir $(RAGEL_TAR)) && \
```

Comment:
Addressing consistency nits.

There isn't any official binary. Ragel 2 is GPL 2 so I'm pretty sure we can't build and distribute our own binaries. (There's an exception about the generated code in the license.)